workflow/activity_options.go

```diff
@@ -71,3 +74,8 @@ func WithHeartbeatTimeout(ctx Context, d time.Duration) Context {
 func WithWaitForCancellation(ctx Context, wait bool) Context {
 	return internal.WithWaitForCancellation(ctx, wait)
 }
+
+// WithRetryPolicy adds wait for the cacellation to the copy of the context.
```

Comment:
```suggestion
// WithRetryPolicy adds wait for the cancellation to the copy of the context.
```

Reply:
You fixed a typo in a copy paste error. 😆